ext/intl/spoofchecker/spoofchecker_main.c

```diff
@@ -28,9 +28,11 @@ PHP_METHOD(Spoofchecker, isSuspicious)
 	zval *error_code = NULL;
 	SPOOFCHECKER_METHOD_INIT_VARS;
 
-	if (FAILURE == zend_parse_parameters(ZEND_NUM_ARGS(), "s|z", &text, &text_len, &error_code)) {
-		RETURN_THROWS();
-	}
+	ZEND_PARSE_PARAMETERS_START(1, 2)
+		Z_PARAM_STRING(text, text_len)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_ZVAL_OR_NULL(error_code)
```

Comment:
Previously this was just 'z', i.e. Z_PARAM_ZVAL, but now this is "OR_NULL". Since null can't be passed by reference it's probably more correct to use `Z_PARAM_ZVAL` ? Same comment for further down below.

Reply:
True, I ll go by it in my next PR.